.github/workflows/library.yml

```diff
@@ -359,7 +359,7 @@ jobs:
           git config --local user.email "propagate-bot@pymedphys.com"
           git config --local user.name "PyMedPhys Propagate Bot"
 
-          git fetch origin $BRANCH
+          git fetch origin $BRANCH || git remote add upstream_fork "https://github.com/${REPO}.git" && git fetch upstream_fork
```

Comment:
Oh, that's cool :slightly_smiling_face:. Thanks for fixing that :slightly_smiling_face:

Reply:
I was also trying to fix the failing tests.

The problem of _Library / Tests (windows, docs) (pull_request)_ seems related to poetry with python 3.9, and should be fixed by [these changes](https://github.com/python-poetry/poetry/pull/5645) when they will be available.